web_accounts_list.json

```diff
@@ -1440,6 +1440,18 @@
          "category" : "travel",
          "valid" : true
       },
+      {
+         "name" : "Slack",
+         "check_uri" : "https://{account}.slack.com",

```

Comment:
I don't think this would look for a user named "account" but more a slack group with that name.